superset/db_engine_specs/base.py

```diff
@@ -193,7 +193,7 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
     engine_aliases: Set[str] = set()
     drivers: Dict[str, str] = {}
     default_driver: Optional[str] = None
-    allow_ssh_tunneling = False
+    allow_ssh_tunneling = True
```

Comment:
This is kind of a nit, but I think your first inclination to make the default value `False` was a good one and follows convention. Can we instead flip the behavior, so something like `disable_ssh_tunneling = False` (or some name that is shorter. :)

Reply:
Oh! Ok, yeah, makes sense, let me change it